### DISCUSSION TITLE
There is no fill metodth?

### DISCUSSION THREAD

**Discussion by gavalierm** [Makuna/NeoPixelBus#672] [Q&A] [answered]
I use Adafruit neopixel library a trying find something more snap. But i use a lot function named "fill". Because i use in the most of the cases fill all pixes (7 pixel shield).

Yes i can write custom metodth to fill from 0 to numPixels but can not be in library natively?

Thank you.

**Comment by Makuna** [comment, marked as answer]
(moved to discussions/Q&A as this isn't a issue/bug)

[NeoPixelBus API in Wiki](https://github.com/Makuna/NeoPixelBus/wiki/NeoPixelBus-object-API)

The method you are looking for is [ClearTo()](https://github.com/Makuna/NeoPixelBus/wiki/NeoPixelBus-object-API#void-cleartocolorobject-color).  There are two, one for all and one for a subset of pixels.
